packages/atomic/src/components/atomic-result-list-v1/atomic-result-list.pcss

```diff
@@ -82,18 +82,18 @@ atomic-result-list-v1 {
     }
 
     @screen desktop-only {
-      @apply atomic-list-with-dividers;
```

Comment:
Just wondering, will tailwind support `@apply` inside `@screen` in a future version of JIT? or is it expected to be permanently non-compatible?

Reply:
I can't find any information on it, unfortunately.